config/v1/types_node.go

```diff
@@ -46,6 +46,14 @@ type NodeSpec struct {
 	// the status and corresponding reaction of the cluster
 	// +optional
 	WorkerLatencyProfile WorkerLatencyProfileType `json:"workerLatencyProfile,omitempty"`
+
+	// MinimumKubeletVersion is the lowest version of a kubelet that can meaningfully join the cluster.
```

Comment:
can we be more specific about what "meaningfully" entitle and about "deny (most)"?

Reply:
yeah I added more details